Makefile

```diff
@@ -99,3 +110,143 @@ verify-shellscripts-initramfs:
+	cat gen_cmdline.sh | sed \
+		-e '/#BEGIN FEATURES parse_cmdline()/ r out/temp/parse_cmdline' \
+		-e '/#BEGIN FEATURES longusage()/ r out/temp/longusage' \
+		> out/gen_cmdline.sh
+
+out/gen_initramfs.sh: out/build-config
+	cat gen_initramfs.sh | sed \
+		-e '/#BEGIN FEATURES append_base_layout()/ r out/temp/append_base_layout' \
+		-e '/#BEGIN FEATURES create_initramfs()/ r out/temp/create_initramfs' \
+		-e '/#BEGIN FEATURES initramfs_append_func/ r out/temp/initramfs_append_func' \
+		> out/gen_initramfs.sh
+
+out/gen_determineargs.sh: out/build-config
+	cat gen_determineargs.sh | sed \
+		-e '/#BEGIN FEATURES determine_real_args()/ r out/temp/determine_real_args' \
+		> out/gen_determineargs.sh
+
+out/software.sh:
+	cat defaults/software.sh | sed \
+		-e "s:VERSION_BCACHE_TOOLS:${VERSION_BCACHE_TOOLS}:"\
+		-e "s:VERSION_BOOST:${VERSION_BOOST}:"\
+		-e "s:VERSION_BTRFS_PROGS:${VERSION_BTRFS_PROGS}:"\
+		-e "s:VERSION_BUSYBOX:${VERSION_BUSYBOX}:"\
+		-e "s:VERSION_COREUTILS:${VERSION_COREUTILS}:"\
+		-e "s:VERSION_CRYPTSETUP:${VERSION_CRYPTSETUP}:"\
+		-e "s:VERSION_DMRAID:${VERSION_DMRAID}:"\
+		-e "s:VERSION_DROPBEAR:${VERSION_DROPBEAR}:"\
+		-e "s:VERSION_EUDEV:${VERSION_EUDEV}:"\
+		-e "s:VERSION_EXPAT:${VERSION_EXPAT}:"\
+		-e "s:VERSION_E2FSPROGS:${VERSION_E2FSPROGS}:"\
+		-e "s:VERSION_FUSE:${VERSION_FUSE}:"\
+		-e "s:VERSION_GPG:${VERSION_GPG}:"\
+		-e "s:VERSION_HWIDS:${VERSION_HWIDS}:"\
+		-e "s:VERSION_ISCSI:${VERSION_ISCSI}:"\
+		-e "s:VERSION_JSON_C:${VERSION_JSON_C}:"\
+		-e "s:VERSION_KMOD:${VERSION_KMOD}:"\
+		-e "s:VERSION_LIBAIO:${VERSION_LIBAIO}:"\
+		-e "s:VERSION_LIBGCRYPT:${VERSION_LIBGCRYPT}:"\
+		-e "s:VERSION_LIBGPGERROR:${VERSION_LIBGPGERROR}:"\
+		-e "s:VERSION_LIBXCRYPT:${VERSION_LIBXCRYPT}:"\
+		-e "s:VERSION_LVM:${VERSION_LVM}:"\
+		-e "s:VERSION_LZO:${VERSION_LZO}:"\
+		-e "s:VERSION_MDADM:${VERSION_MDADM}:"\
+		-e "s:VERSION_MULTIPATH_TOOLS:${VERSION_MULTIPATH_TOOLS}:"\
+		-e "s:VERSION_POPT:${VERSION_POPT}:"\
+		-e "s:VERSION_STRACE:${VERSION_STRACE}:"\
+		-e "s:VERSION_THIN_PROVISIONING_TOOLS:${VERSION_THIN_PROVISIONING_TOOLS}:"\
+		-e "s:VERSION_UNIONFS_FUSE:${VERSION_UNIONFS_FUSE}:"\
+		-e "s:VERSION_USERSPACE_RCU:${VERSION_USERSPACE_RCU}:"\
+		-e "s:VERSION_UTIL_LINUX:${VERSION_UTIL_LINUX}:"\
+		-e "s:VERSION_XFSPROGS:${VERSION_XFSPROGS}:"\
+		-e "s:VERSION_XZ:${VERSION_XZ}:"\
+		-e "s:VERSION_ZLIB:${VERSION_ZLIB}:"\
+		-e "s:VERSION_ZSTD:${VERSION_ZSTD}:"\
+		> out/software.sh
+
+out/genkernel: out/genkernel.conf out/gen_cmdline.sh \
+out/gen_initramfs.sh out/gen_determineargs.sh out/software.sh
+
+	cp gen_arch.sh out/gen_arch.sh
+	cp gen_bootloader.sh out/gen_bootloader.sh
+	cp gen_compile.sh out/gen_compile.sh
+	cp gen_configkernel.sh out/gen_configkernel.sh
+	cp gen_funcs.sh out/gen_funcs.sh
+	cp gen_moddeps.sh out/gen_moddeps.sh
+	cp gen_package.sh out/gen_package.sh
+	cp gen_worker.sh out/gen_worker.sh
+	cp path_expander.py out/path_expander.py
+	cp genkernel out/genkernel
```

Comment:
mismatch of the deps vs what is copied. 
All the names are the same as well, so this should be:
```
out/%: %
  cp $< $@
out/genkernel: $(addprefix out/,$(DEPS))
```